docs/orbs-cd.md

```diff
@@ -0,0 +1,69 @@
+# CD Techniques
+
+_Placeholder for more information regarding CD_
+
+## The `circleci/orb-tools` orb:
+
+In order to aid you in your CD efforts we have put together an orb-tools orb you can
+reference while building your own orb.
+
```

Comment:
Please add a sentence that says "`orb-tools` is a shortcut, but not a pre-requisite to orb publishing. All publishing activity can be conducted directly from the (`circleci` CLI)[LINK HERE]